examples/async_io/CMakeLists.txt

```diff
@@ -7,7 +7,11 @@ set(example_programs
     async_io_simple
     async_io_action
     async_io_external
-    async_io_low_level)
+    async_io_low_level
+		async
+		then
+		wait
+		wait_all)
```

Comment:
1. Change tabs to spaces.
2. Move your examples into a separate directory instead of reusing another unrelated one.